GOVERNANCE.md

```diff
@@ -244,7 +277,153 @@ The current NumFOCUS Subcommittee consists of:
+explicit about who are core contributors, especially for people interested
+in joining the project to reach out, but also for organizations and meet-ups
+to ask for speakers.
+
+Regarding the reaching out, we do have the project mail and some people
+may not desire being contacted privately. I also think we should have some
+rules or guidance on this here and/or in the community/contributing pages.-->
+
+## Recurring Contributors
+Recurring Contributors are those individuals who can participate from
+The Project private communication channels.
+
+### Recurring Contributor membership
+Recurring Contributors can nominate any Contributor to participate in the
+Project private communication channels and become a Recurring Contributor.
+For the nomination to go forward, it has to be ratified by
+the Steering Council and the BDFL.
+For a nomination to be rejected, clear reasoning behind the decision must be
+shared with the rest of the team. People whose nomination has been rejected can
+be nominated at any time again in the future, three months after the previous
+nomination at the earliest.
+
+#### Current Recurring Contributors
+
+- everyone on slack! This may be too long so it could be a good idea to
+format as a table instead or to put within a `<details>` tag so
+it's hidden by default while still available publicly
+
+## Conflict of interest
+
+It is expected that the BDFL, Council Members and Contributors will be
+employed at a wide range of companies, universities and non-profit organizations.
+Because of this, it is possible that Members will have conflict of interests.
+Such conflict of interests include, but are not limited to:
+
+- Financial interests, such as investments, employment or contracting work,
+  outside of The Project that may influence their work on The Project.
+- Access to proprietary information of their employer that could potentially
+  leak into their work with the Project.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused themselves for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+## Team Organization
+<!-- Open questions
+
+1. do we want to give commit rights to core contributors to some repositories
+only based on the team they are part of?
+
+(quick discussion with myself - Oriol) With pymc3 being the source of the
+website and a large part of the docs, both members of developer and documentation
+team need to have commit rights there, so it would basically mean that
+devs would not have commit rights to pymc-examples and/or resources repo.
+**However**, this would become much more relevant and sensible if we had more
+teams! (see next question).
+
+2. do we want to create/formalize other teams?
```

Comment:
There are not than many GitHub permission levels and not so many repositories for which we'd need teams anyway. It would quickly become redundant.

Reply:
New teams might not have anything to do with github. I personally would wait for teams to start organically, but I think it's good for people to know it's possible to do. See the details dropdown in the pr description for an example of possible evolution in a few years